File: LC-1365/LC-1365-1.cpp
#include <vector>

using namespace std;

class Solution {
public:
    /**
     * Concepts: Naive Approach
     */
    vector<int> smallerNumbersThanCurrent(vector<int>& nums) {
        vector<int> smallerNumbers(nums.size(), 0);
        for (int i = 0; i < nums.size(); ++i) {
            for (int j = 0; j < nums.size(); ++j) {
                if (i != j && nums[j] < nums[i]) {
                    ++smallerNumbers[i];
                }
            }
        }
        return smallerNumbers;
    }
};

File: LC-1365/LC-1365-1.js
/**
 * @param {number[]} nums
 * @return {number[]}
 */
var smallerNumbersThanCurrent = function (nums) {
  /**
   * Concepts: Naive Approach
   */

  let smallerNumbers = Array(nums.length).fill(0)
  for (let i = 0; i < nums.length; ++i) {
    for (let j = 0; j < nums.length; ++j) {
      if (i !== j && nums[j] < nums[i]) ++smallerNumbers[i]
    }
  }
  return smallerNumbers
}
